android/src/main/java/com/neuroidreactnativesdk/NeuroidReactnativeSdkModule.kt

```diff
@@ -47,6 +47,7 @@ class NeuroidReactnativeSdkModule(reactContext: ReactApplicationContext) :
     
     @ReactMethod
     fun start() {
+         NeuroID.getInstance()?.setIsRN()
```

Comment:
formatting...

Reply:
I am not sure how this is happening because in my editor it looks lined up and if I adjust it then it looks like the spacing is wrong